log/logfmt_logger.go

```diff
@@ -0,0 +1,29 @@
+package log
+
+import (
+	"io"
+
+	"gopkg.in/logfmt.v0"
+)
+
+type logfmtLogger struct {
+	w io.Writer
+}
+
+// NewLogfmtLogger returns a basic logger that encodes keyvals as simple "k=v"
+// pairs to the Writer.
+func NewLogfmtLogger(w io.Writer) Logger {
+	return &logfmtLogger{w}
+}
+
```

Comment:
golint doesn't complain about it. Curious why this is, I looked at the lint code and learned that it doesn't complain because the receiver type (`logfmtLogger`) is not exported. Because the receiver type is not exported, the `Log` method does not appear in the godoc.

We could add a comment such as

    // Log implements the Logger interface.

But that seems like noise in this case.